packages/design-system/src/styles/components/_LabelMask.scss

```diff
@@ -0,0 +1,30 @@
+@import '../settings/index.scss';
+
+// Changing DOM order of label mask and input
+// Maintaining expected visual order
+.ds-c-label-mask {
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.ds-c-field + .ds-c-label-mask__mask {
+  font-family: Menlo, Consolas, Monaco, Liberation Mono, Lucida Console, monospace;
```

Comment:
[question] Is this a font that is used elsewhere in the design system?

Reply:
Oh you're right, looks like there's a `$font-monospace` being downloaded/used for our docs site (using webfonts) that could be used here. That's a bummer - the OS fonts work just as good here without the additional file download 😞 

Updating now.

Reply:
OH! Seems like this font rule is specific to our docs site and can't be used in our component library! Disregard my previous comment

Reply:
Added a comment to the Jira ticket pointing this out.